### PR TITLE
Fix for After account switch user is always taken to manage business;…

### PR DESCRIPTION
**Pull request by saravanpa-aot**
…user should stay in the same page

*Issue #:*
https://github.com/bcgov/entity/issues/<Put the github issue number here>

*Description of changes:*

Now when account is switched , user is always taken to manage business.This change to put the original behaviour back

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/views/auth/AcceptInviteView.vue [hunk 1]
```diff
@@ -36,6 +36,8 @@ export default class AcceptInviteView extends Mixins(NextPageMixin) {
   private orgStore = getModule(OrgModule, this.$store)
   private userStore = getModule(UserModule, this.$store)
   private readonly acceptInvitation!: (token: string) => Promise<Invitation>
+  private readonly syncMembership!: (currentAccountId: string) => Promise<Member>
```
Comment (sutherlanda) [inline auth-web/src/views/auth/AcceptInviteView.vue]: The actual function in the `org` module takes a number, not a string for the id.

File: auth-web/src/views/auth/AcceptInviteView.vue [hunk 2]
```diff
@@ -61,7 +63,9 @@ export default class AcceptInviteView extends Mixins(NextPageMixin) {
       } else {
         const invitation = await this.acceptInvitation(this.token)
         ConfigHelper.addToSession(SessionStorageKeys.CurrentAccount, JSON.stringify({ id: invitation.membership[0].org.id }))
+        await this.syncMembership(invitation.membership[0].org.id.toString())
```
Comment (sutherlanda) [inline auth-web/src/views/auth/AcceptInviteView.vue]: Once you changes the function signature, you don't have to apply `toString`

Comment (sutherlanda) [inline auth-web/src/views/auth/AcceptInviteView.vue]: Should we use some optional chaining here to handle empty memberships?

Comment (saravanpa-aot) [reply auth-web/src/views/auth/AcceptInviteView.vue]: excellent catch..thanks

Comment (saravanpa-aot) [reply auth-web/src/views/auth/AcceptInviteView.vue]: Done

File: auth-web/src/views/auth/AcceptInviteView.vue (hunk 1 above)
Comment (saravanpa-aot) [reply auth-web/src/views/auth/AcceptInviteView.vue]: Done

**Pull request merged**